lib/spack/spack/mirror.py

```diff
@@ -120,28 +120,29 @@ def suggest_archive_basename(resource):
 
 def create(path, specs, **kwargs):
     """Create a directory to be used as a spack mirror, and fill it with
-       package archives.
-
-       Arguments:
-         path    Path to create a mirror directory hierarchy in.
-         specs   Any package versions matching these specs will be added
-                 to the mirror.
-
-       Keyword args:
-         no_checksum:  If True, do not checkpoint when fetching (default False)
-         num_versions: Max number of versions to fetch per spec,
-                       if spec is ambiguous (default is 0 for all of them)
-
-       Return Value:
-         Returns a tuple of lists: (present, mirrored, error)
-         * present:  Package specs that were already present.
-         * mirrored: Package specs that were successfully mirrored.
-         * error:    Package specs that failed to mirror due to some error.
-
-       This routine iterates through all known package versions, and
-       it creates specs for those versions.  If the version satisfies any spec
-       in the specs list, it is downloaded and added to the mirror.
-    """
+    package archives.
+
+    Arguments:
+        path: Path to create a mirror directory hierarchy in.
+        specs: Any package versions matching these specs will be added \
+            to the mirror.
+
+    Keyword args:
+        no_checksum: If True, do not checkpoint when fetching (default False)
+        num_versions: Max number of versions to fetch per spec, \
+            if spec is ambiguous (default is 0 for all of them)
+
+    Return Value:
+        Returns a tuple of lists: (present, mirrored, error)
+
+        * present:  Package specs that were already present.
+        * mirrored: Package specs that were successfully mirrored.
+        * error:    Package specs that failed to mirror due to some error.
+
+    This routine iterates through all known package versions, and
+    it creates specs for those versions.  If the version satisfies any spec
+    in the specs list, it is downloaded and added to the mirror."""
+
```

Comment:
@alalazo The docstring changes you noticed were a result of how Sphinx handles docstrings. Take a look at the [docstring for zsh](https://github.com/LLNL/spack/blob/develop/var/spack/repos/builtin/packages/zsh/package.py#L29). Now take a look at Spack's [documentation on the zsh package](http://software.llnl.gov/spack/package_list.html#zsh). You'll notice that the newline at the start of the docstring caused improper indentation in the resulting .rst file, causing the documentation to look funny.

I tried to only impact the docstrings that were broken by this. I don't think the trailing triple quotes actually matter, but the leading triple quotes definitely can't be followed by a newline. I don't think this will be too hard to enforce, as the documentation tests will fail when anyone tries to put a newline there.

As far as the trailing triple quotes go, I don't really care either way. Would you like me to change them back?